src/mumble/Settings.h

```diff
@@ -188,7 +188,7 @@ struct OverlaySettings {
 struct Settings {
 	enum AudioTransmit { Continuous, VAD, PushToTalk };
 	enum VADSource { Amplitude, SignalToNoise };
-	enum LoopMode { None, Local, Server };
+	enum LoopMode { None, LocalOnly, ServerOnly, LocalRegular, ServerRegular };
```

Comment:
As before, I don't think "regular" is descriptive - should be adapted in unison with the special whisper target.